src/commands/general/temporaryCommand.ts

```diff
@@ -0,0 +1,177 @@
+import { Colors, EmbedField } from 'discord.js';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory, Colors as FBWColors } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { getConn } from '../../lib/db';
+import TemporaryCommand from '../../lib/schemas/temporaryCommandSchema';
+
+const helpEmbed = (evokedCommand: String) => makeEmbed({
+    title: 'Temporary Command - Help',
+    description: 'A command to run temporary commands.',
+    fields: [
+        {
+            name: 'Run a Temporary Command',
+            value: makeLines([
+                'To run a temporary command, run the following bot command: ',
+                `\`${evokedCommand} <command>\`.`,
+                '`command`: The command to run, needs to be an exact match.',
+                'Example:',
+                `\`${evokedCommand} good-news-everyone\``,
+                '\u200B',
+            ]),
+            inline: false,
+        },
+        {
+            name: 'List existing Temporary Commands',
+            value: makeLines([
+                'To list the existing temporary commands, run the following bot command: ',
```

Comment:
```suggestion
                'To list the existing temporary commands, use the following bot command: ',
```

For continuity if you agree with above